ansible_ai_connect/main/settings/base.py

```diff
@@ -612,3 +612,9 @@ def is_ssl_enabled(value: str) -> bool:
 CHATBOT_DEFAULT_PROVIDER = os.getenv("CHATBOT_DEFAULT_PROVIDER")
 CHATBOT_DEFAULT_MODEL = os.getenv("CHATBOT_DEFAULT_MODEL")
 # ==========================================
+
+# =============================================
+# Playbook Generation/Explanation for "on-prem"
+# ---------------------------------------------
+ENABLE_PLAYBOOK_ENDPOINT = os.getenv("ENABLE_PLAYBOOK_ENDPOINT", "False").lower() == "true"
+# =============================================
```

Comment:
Looks like default version is `False`, it means PR should have a note in **Production deployment** section of this PR

Reply:
Ah, well, @hasys at the moment I only check the flag for "on prem".. so the endpoints are *always* enabled for SaaS.

I asked @jameswnl to clarify the JIRA Acceptance Criteria.. easy enough to use for both SaaS and "on prem".

See `#team-wisdom-engineering`.

Reply:
@hasys @jameswnl I've changed the PR to use the new environment variable for both SaaS and "on prem".

There is a corresponding [PR](https://github.com/ansible/ansible-wisdom-ops/pull/1096) for `ansible-wisdom-ops` to enable it for SaaS.

Local development (using `development.py` settings) also [enables](https://github.com/ansible/ansible-ai-connect-service/pull/1375/files#diff-30c4274265178ee8e8dfb34bd09feacd9d166040ee15fe3d9be42cf923996897R75) it by default.